app/src/main/res/layout/topic_practice_skill_view.xml

```diff
@@ -10,7 +10,7 @@
 
     <variable
```

Comment:
Rename file to `topic_practice_subtopic`

Reply:
Done.